kasa/smart/smartdevice.py

```diff
@@ -35,6 +41,16 @@
 # same issue, homekit perhaps?
 NON_HUB_PARENT_ONLY_MODULES = [DeviceModule, Time, Firmware, Cloud]
 
+DELAY_UPDATE_MODULE_SECONDS = {
+    Cloud: 60,
+    Firmware: 60 * 60 * 24,
+    # A memory leak in the P100 with the Led module so we delay it.
+    Led: 60 * 60,
+    LightPreset: 60,
+    LightEffect: 60,
+    LightTransition: 60,
+}
```

Comment:
It's better to define this inside the corresponding module, perhaps as a class constant like `MINIMUM_UPDATE_INTERVAL`?

Reply:
Nice idea, that's nicer and is now done